docs/source/en/api/pipelines/stable_unclip.mdx

```diff
@@ -64,6 +60,13 @@ response = requests.get(url)
 init_image = Image.open(BytesIO(response.content)).convert("RGB")
 init_image = init_image.resize((768, 512))
 
+images = pipe(init_image).images
+images[0].save("fantasy_landscape.png")
+```
+
+Optionally, you can also pass a prompt to `pipe` such as:
+
+```python 
 prompt = "A fantasy landscape, trending on artstation"
 
 images = pipe(prompt, init_image).images
```

Comment:
```suggestion
images = pipe(init_image).images
```

Reply:
Here we are telling the readers that they can also pass a prompt. Passing a prompt is not necessary is something we covered in the earlier section. See here: https://github.com/huggingface/diffusers/pull/2815/files#diff-0bec8187fde802d6ffcdfbad61c1d944e0a29179e185a6214514fb08ced13725R63

Reply:
You're right - thanks!